server/app/routes/v1/grids_api.rb

```diff
@@ -127,8 +127,16 @@ class GridsApi < Roda
             end
             unless r['services'].nil?
               services = r['services'].split(',').map do |service|
-                @grid.grid_services.find_by(name: service).try(:id)
-              end.delete_if{|s| s.nil?}
+                if service.include?('/')
+                  stack_name, service_name = service.split('/', 2)
+                  stack_id = @grid.stacks.where(name: stack_name).first.try(:id)
+                  if stack_id && service_name
+                    @grid.grid_services.where(stack_id: stack_id, name: service_name).first.try(:id)
+                  end
+                else
+                  @grid.grid_services.find_by(name: service).try(:id)
```

Comment:
If the service name does not include a `/`, then this should probably be fixed to only the matching stackless service, not the first service similarly named service in any stack? aka something like `@grid.stacks.find_by(name: Stack::NULL_STACK).grid_services.find_by(name: service)`